andela/wger-prometheus
# #150397733 Add fitbit support

#### What does this PR do?
Add fitbit support
#### Description of Task to be completed?
Enable logged in users with fitbit accounts to add weight logged into their fitbit accounts to their wger accounts
#### How should this be manually tested?
- Create a fitbit account and log your weight
- In wger, select weight overview in the weight drop down
- In the weight overview page click on the options button
- Select import from the drop down menu
- The page should reload and display the weight logged into the fitbit account in the rendered graph and in the table
#### What are the relevant pivotal tracker stories?
#150397733
#### Screenshots
<img width="1440" alt="screen shot 2017-09-15 at 12 35 40" src="https://user-images.githubusercontent.com/13675851/30476462-c3245a1e-9a12-11e7-9b0c-5f7e73504f2d.png">
